iahx/templates/custom/base.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html dir="ltr" lang="{{lang}}" xmlns="http://www.w3.org/1999/xhtml">
+    <head>{% block head %}
+
+        <meta charset="utf-8">
+        <title>{% block title %}{{ texts.SEARCH_HOME }} | {{ texts.BVS_TITLE }}{% endblock %}</title>
+
+        <!-- meta-tags -->
+        <meta http-equiv="Expires" content="-1">
+        <meta http-equiv="pragma" content="no-cache">
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <meta http-equiv="Content-Language" content="{{ lang }}">
+        <meta name="robots" content="all">
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0">
+
+        <meta http-equiv="keywords" content="{{ texts.KEYWORDS }}">
+        <meta http-equiv="description" content="{{ texts.DESCRIPTION }} ">
+
+        <!-- css -->
+        <link rel="stylesheet" href="{{ constant("STATIC_URL") }}css/bootstrap.min.css">
+        <link rel="stylesheet" href="{{ constant("STATIC_URL") }}css/scielo-portal.css">
+        <link rel="stylesheet" href="{{ constant("STATIC_URL") }}css/scielo-print.css" media="print">
+
+
+        <!-- Page icon -->
+        <link rel="shortcut icon" href="{{ constant("STATIC_URL") }}image/favicon.ico">
+
+        <!-- RSS -->
+        <link rel="alternate" type="application/rss+xml" title="RSS" href="index.php?output=rss&amp;site=bvsms&amp;col=main&amp;lang=pt&amp;sort=da+desc">
+
+        <script language="javascript" type="text/javascript">
```

Comment:
Talvez no outro PR já esteja corrigido, mas devemos manter os javascripts no final do html, ver: http://stackoverflow.com/questions/436411/where-is-the-best-place-to-put-script-tags-in-html-markup (Nesse artigo ele defende o uso do atributo async e defer, porém não é suportado por alguns browser o que nos obriga a somente colocar no final do HTML)